### PR TITLE
Update node/npm and ensure we all run the same versions.

### PR DESCRIPTION
**Pull request by dazoakley**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,6 +8,10 @@
     "cypress:ci": "npx cypress run --browser chrome --spec ./cypress/integration/*.feature",
     "cypress:report": "node cypress/cucumber-report.js"
   },
+  "engines": {
+    "node": "16.13.2",
+    "npm": "8.4.0"
+  },
```
Comment (philsegal) [inline package.json:14]: do we need the node version in 2 places? Feel like we had a similar conversation in UI @jonnywyatt , dont recall if we concluded anything

Comment (dazoakley) [reply package.json:14]: This one enforces it...  I can remove the `.node-version` file if you really don't want it in two places (It's in 3 in ui 😂 )

**Review by philsegal — DISMISSED** [review]

**Review by philsegal — APPROVED** [review]

File: .circleci/config.yml [hunk 2]
```diff
@@ -71,27 +89,20 @@ jobs:
...
+      name: hmpps/node
+      tag: 16.13-browsers
     resource_class: large
-    docker:
-      - image: cypress/included:9.2.1
-    working_directory: /mnt/ramdisk
     environment:
       ENVIRONMENT: <<pipeline.parameters.e2e_environment>>
     steps:
+      - browser-tools/install-chrome
```
Comment (jonnywyatt) [inline .circleci/config.yml:99]: is it more efficient to use the cypress/included image, to save re-installing these each time, unless they're cached?

Comment (dazoakley) [reply .circleci/config.yml:99]: They're cached - but the main issue with the `cypress/included` image is you have no control over the version of node/npm in use - it's quite old...

**Pull request merged**